samples/web/content/apprtc/room.py

```diff
@@ -65,6 +65,27 @@ def has_room(host, room_id):
   room = memcache_client.get(key)
   return room is not None
 
+def remove_room(host, room_id, client_id):
+  memcache_client = memcache.Client()
+  key = get_memcache_key_for_room(host, room_id)
+  room = memcache_client.get(key)
+  
+  if room is None:
+    logging.warning('Can\'t remove room ' + room_id +
+        ' because it doesn\'t exist, client_id: ' + client_id)
+    return constants.RESPONSE_INVALID_ROOM
+
+  if not room.is_client_allowed(client_id):
+    logging.warning('Can\'t remove room ' + room_id +
+        ' because room does not allow client ' + client_id)
+    return constants.RESPONSE_INVALID_CALLEE
```

Comment:
Also need to check that the room does not already have two clients, i.e. call already accepted.